BitmovinAnalyticsCollector/Classes/Collector/data/EventData.swift

```diff
@@ -60,9 +60,18 @@ public class EventData: Codable {
     var platform: String = "iOS"
     #elseif os(tvOS)
     var platform: String = "tvOS"
+    #elseif os(watchOS)
+    var platform: String = "watchOS"
+    #elseif os(macOS)
+    var platform: String = "macOS"
+    #elseif os(Linux)
```

Comment:
👍 although not strictly necessary as I don't think these are actually possible right now

Reply:
There are impressions in the database that don't have the platform set for iOS - I want to make sure we don't miss them, so we have at least `unknown` there, and can debug where the problem is.